assets/CET_to_py.py

```diff
@@ -144,103 +150,142 @@ def all_original_names(group=None, not_group=None, only_aliased=False, only_CET=
-  linear_ternary_red_0_50_c52                     = 'kr',
-  rainbow_bgyr_35_85_c73                          = 'rainbow',
-  glasbey_bw_minc_20                              = 'glasbey',
-  glasbey_bw_minc_20_minl_30                      = 'glasbey_light',
-  glasbey_bw_minc_20_maxl_70                      = 'glasbey_dark',
-  glasbey_bw_minc_20_hue_330_100                  = 'glasbey_warm',
-  glasbey_bw_minc_20_hue_150_280                  = 'glasbey_cool',
+    circle_mgbm_67_c31                              = ['cyclic_isoluminant', 'colour'],
+    cyclic_grey_15_85_c0                            = ['cyclicgrey'],
+    cyclic_mygbm_30_95_c78                          = ['phase4'],
+    cyclic_mygbm_30_95_c78_s25                      = ['colorwheel'],
+    cyclic_wrwbw_40_90_c42                          = ['phase2'],
+    diverging_bkr_55_10_c35                         = ['bkr'],
+    diverging_bky_60_10_c30                         = ['bky'],
+    diverging_bwg_20_95_c41                         = ['bwg'],
+    diverging_bwr_20_95_c54                         = ['bwra'],
+    diverging_bwr_40_95_c42                         = ['bwr', 'coolwarm'],  # mpl
+    diverging_gkr_60_10_c40                         = ['gkr'],
+    diverging_gwr_55_95_c38                         = ['gwr'],
+    diverging_gwv_55_95_c39                         = ['gwv'],
+    diverging_linear_bjr_30_55_c53                  = ['bjr'],
+    diverging_linear_bjy_30_90_c45                  = ['bjy', 'divbjy'],
+    diverging_protanopic_deuteranopic_bwy_60_95_c32 = ['bwy'],
+    diverging_rainbow_bgymr_45_85_c67               = ['rainbow3'],
+    diverging_tritanopic_cwr_75_98_c20              = ['cwr'],
+    glasbey_bw_minc_20                              = ['glasbey'],
+    glasbey_bw_minc_20_hue_150_280                  = ['glasbey_cool'],
+    glasbey_bw_minc_20_hue_330_100                  = ['glasbey_warm'],
+    glasbey_bw_minc_20_maxl_70                      = ['glasbey_dark'],
+    glasbey_bw_minc_20_minl_30                      = ['glasbey_light'],
+    isoluminant_cgo_80_c38                          = ['isolum'],
+    linear_bgy_10_95_c74                            = ['bgy'],
+    linear_bgyw_15_100_c68                          = ['bgyw'],
+    linear_blue_5_95_c73                            = ['kbc', 'linear_kbc_5_95_c73'],
+    linear_blue_95_50_c20                           = ['depth', 'blues'],  # mpl
+    linear_bmw_5_95_c89                             = ['bmw'],
+    linear_bmy_10_95_c78                            = ['bmy'],
+    linear_gow_60_85_c27                            = ['geographic'],
+    linear_gow_65_90_c35                            = ['geographic2'],
+    linear_green_5_95_c69                           = ['kgy', 'linear_kgy_5_95_c69'],
+    linear_grey_0_100_c0                            = ['grey', 'gray'],  # mpl
+    linear_grey_10_95_c0                            = ['reducedgrey', 'dimgray'],
```

Comment:
Not sure what 'reducedgrey' adds as an alias here; seems like that will just make the space of names less usable for e.g. GUI colormap selector widgets, with several indistinguishable options.

Reply:
Same comment as above. `reducedgrey` is automatically imported from `colorcet.m`.